doc/documentation-main.html

```diff
@@ -1793,6 +1793,52 @@ <h4><a class="section" indent=".." name="rerunning">Rerunning failed tests</a></
 methods so that you are guaranteed to run the methods that failed without any 
 SKIP failures.</p>
 
+Sometimes you might also want TestNG to automatically retry a test whenever it fails. In those situations you can make use of a retry analyzer.
```

Comment:
Sometimes, you might want TestNG to automatically retry a test whenever it fails. In those situations, you can use a retry analyzer. When you bind a retry analyzer to a test, TestNG automatically invokes the retry analyzer to determine if TestNG can retry a test case again in an attempt to see if the test that just fails now passes.

Here is how you use a retry analyzer:

Reply:
Amended the commit and fixed as suggested